### PR TITLE
Add documentation for new opnsense integration

### PR DESCRIPTION
**Pull request by mtreinish**
**Description:**

This commit adds documentation for the opnsense device tracker which is
being added in home-assistant/home-assistant#26834.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26834

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/opnsense.markdown [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "OPNsense"
+description: "Instructions on how to configure OPNsense integration"
+logo: opnsense.png
+ha_category:
+  - Hub
+  - Presence Detection
+ha_release: 0.100
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/opnsense.markdown]: Please remove the `redirect_from`

File: source/_components/opnsense.markdown [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+title: "OPNsense"
+description: "Instructions on how to configure OPNsense integration"
+logo: opnsense.png
+ha_category:
+  - Hub
+  - Presence Detection
+ha_release: 0.100
```
Comment (klaasnicolaas) [inline source/_components/opnsense.markdown]:
```suggestion
ha_release: "0.100"
```

File: source/_components/opnsense.markdown [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Where the `api_key` and `api_secret` values are aquired from your OPNsense
+router using the web interface. For more inforation on this procedure refer
+to the OPNsense [documentation](https://docs.opnsense.org/development/how-tos/api.html#creating-keys)
+
+{% configuration %}
+url:
+  description: The url for the OPNsense api endpoint of your router
+  type: string
+  required: true
+  default: None
```
Comment (klaasnicolaas) [inline source/_components/opnsense.markdown]: If there is no default in the python code, you can leave it away.

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/opnsense.markdown (hunk 1 above)
Comment (mtreinish) [reply source/_components/opnsense.markdown]: Done in https://github.com/home-assistant/home-assistant.io/pull/10427/commits/dc933232ab654d79a27cc0a16f8ca13b16196d46

File: source/_components/opnsense.markdown (hunk 3 above)
Comment (mtreinish) [reply source/_components/opnsense.markdown]: Done in https://github.com/home-assistant/home-assistant.io/pull/10427/commits/dc933232ab654d79a27cc0a16f8ca13b16196d46

File: source/_components/opnsense.markdown [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+{% configuration %}
+url:
+  description: The url for the OPNsense api endpoint of your router
+  type: string
+  required: true
+api_key:
+  description: The API key used to authenticate with your OPNsense API endpoint
+  type: string
+  required: true
+  default: None
```
Comment (klaasnicolaas) [inline source/_components/opnsense.markdown]: Also here

File: source/_components/opnsense.markdown [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+api_secret:
+  description: The API secret used to authenticate with your OPNsense API endpoint
+  type: string
+  required: true
+  default: None
```
Comment (klaasnicolaas) [inline source/_components/opnsense.markdown]: And here

File: source/_components/opnsense.markdown [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+verify_ssl:
+  description: Set to true to enable the validation of the OPNsense API's SSL
+  type: boolean
+  required: false
+  default: false
+tracker_interfaces:
+  description: List of the OPNsense router's interfaces to use for tracking devices
+  type: list
+  required: false
+  default: None
```
Comment (klaasnicolaas) [inline source/_components/opnsense.markdown]: And here 😉

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/opnsense.markdown (hunk 5 above)
Comment (mtreinish) [reply source/_components/opnsense.markdown]: Fixed in: https://github.com/home-assistant/home-assistant.io/pull/10427/commits/ddaab6b128d0a1c6376d13cebc22ec5cffa1eb22

File: source/_components/opnsense.markdown (hunk 4 above)
Comment (mtreinish) [reply source/_components/opnsense.markdown]: Oops, heh I guess I applied your earlier suggestion a bit too mechanically. Fixed in: https://github.com/home-assistant/home-assistant.io/pull/10427/commits/ddaab6b128d0a1c6376d13cebc22ec5cffa1eb22

File: source/_components/opnsense.markdown (hunk 6 above)
Comment (mtreinish) [reply source/_components/opnsense.markdown]: This one actually sets a default, just in the device tracker: https://github.com/home-assistant/home-assistant/pull/26834/files#diff-d7915733a6a8b679b376fc58c0b252caR14 I've updated this to reflect that in: https://github.com/home-assistant/home-assistant.io/pull/10427/commits/ddaab6b128d0a1c6376d13cebc22ec5cffa1eb22

**Comment by frenck** [comment]
Rebased the PR onto the latest `next` branch, moved the Markdown file to the new location in the repository.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` needs an update before merging this one in.

**Review by frenck — APPROVED** [review]

**Pull request merged**